src/client/components/AdditionalProductCard.tsx

```diff
@@ -126,7 +126,11 @@ export const Section = styled.section({
 export const Header = styled.header({
   display: 'flex',
   flexDirection: 'column',
-  fontSize: '1.175rem',
+  fontSize: '1rem',
+
+  [MEDIUM_SCREEN_MEDIA_QUERY]: {
+    fontSize: '1.25rem',
```

Comment:
What do you think about moving standard font sizes somewhere to constants to make it easier to reuse?